root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,232 @@
+                            list_form.Add(new PositionData(strKey, "null"));
+                        }
+                        else
+                        {
+                            //Add key and value to PositionData
+                            list_form.Add(new PositionData(strKey, CustomEncode.HtmlEncode(form[strKey].ToString())));
+                        }
+                    }
+                    //データバインド
+                    ViewBag.list_form = list_form;
+                }
+
+                // 画面にセッション情報を表示する------------------------------------------
+
+                if (Session != null)
+                {
+                    //foreach
+                    foreach (string strKey in Session)
+                    {
+                        if (Session[strKey] == null)
+                        {
+                            //Add key and value to PositionData
+                            list_session.Add(new PositionData(strKey, "null"));
+                        }
+                        else
+                        {
+                            //Add key and value to PositionData
+                            list_session.Add(new PositionData(strKey, CustomEncode.HtmlEncode(Session[strKey].ToString())));
+                        }
+                    }
+                    //データバインド
+                    ViewBag.list_session = list_session;
+                }
+
+                // セッション情報を削除する------------------------------------------------
+
+                if (sessionAbandonFlag)
+                {
+                    // セッション タイムアウト検出用Cookieを消去
+                    // ※ Removeが正常に動作しないため、値を空文字に設定 ＝ 消去とする
+
+                    // Set-Cookie HTTPヘッダをレスポンス
+                    Response.Cookies.Set(FxCmnFunction.DeleteCookieForSessionTimeoutDetection());
+
+                    try
+                    {
+                        // セッションを消去                       
+                        Session.Abandon();
+                    }
+                    catch (Exception ex)
+                    {
+                        // このカバレージを通過する場合、おそらく起動した画面のパスが間違っている。
+#if DEBUG
+                        string err_str = "このカバレージを通過する場合、おそらく起動した画面のパスが間違っている。";
+                        Debug.WriteLine(err_str);
+                        Debug.WriteLine(ex.Message);
+                        ViewBag.label1Data = CustomEncode.HtmlEncode(err_str);
+                        ViewBag.label2Data = CustomEncode.HtmlEncode(ex.Message);
+#endif
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                // 開発用エラー画面でエラー（無限ループになるのでエラーを潰している）。
+#if DEBUG
+                string err_str = "開発用エラー画面でエラー（無限ループになるのでエラーを潰している）。";
+                Debug.WriteLine(err_str);
+                Debug.WriteLine(ex.Message);
+                ViewBag.label1Data = CustomEncode.HtmlEncode(err_str);
+                ViewBag.label2Data = CustomEncode.HtmlEncode(ex.Message);
+#endif
```

Comment:
@daisukenishino 

To show messages in case of release build, remove the process to show messages from `#if DEBUG` directive.

```cs
#if DEBUG
                string err_str = "開発用エラー画面でエラー（無限ループになるのでエラーを潰している）。";
                Debug.WriteLine(err_str);
                Debug.WriteLine(ex.Message);
#endif
                ViewBag.label1Data = CustomEncode.HtmlEncode(err_str);
                ViewBag.label2Data = CustomEncode.HtmlEncode(ex.Message);
```